khelo/lib/ui/flow/tournament/match_selection/match_scheduler.dart

```diff
@@ -541,4 +572,37 @@ class MatchScheduler {
       MapEntry(newGroup, {1: []})
     ]);
   }
+
+  TeamModel? scheduleSingleBracketTeams(
+    List<MatchModel> matches,
+    List<TeamModel> teams,
+    bool isWinBracket,
+  ) {
+    final finishedMatches =
+        matches.where((element) => element.match_status == MatchStatus.finish);
+
+    final loserTeams = finishedMatches.map((e) => e.teams
+        .firstWhere((element) => element.team_id != e.matchResult!.teamId)
+        .team);
+    final unFinishedMatches =
+        matches.where((element) => element.match_status != MatchStatus.finish);
+    teams.removeWhere((team) =>
+        loserTeams.contains(team) ||
+        unFinishedMatches
+            .any((element) => element.teams.map((e) => e.team).contains(team)));
+
+    final chunks = createTeamPairsForKnockout(teams);
```

Comment:
Rename the function name according to common usage if we are using it in multiple places.